Resources/Prototypes/Entities/Clothing/Eyes/hud.yml

```diff
@@ -216,3 +216,53 @@
   suffix: Syndicate
   components:
   - type: ShowSecurityIcons
+
+- type: entity
+  parent: ClothingEyesBase
+  id: ClothingEyesEyepatchHudMedical
+  name: medical hud eyepatch
+  description: A heads-up display that scans the humanoids in view and provides accurate data about their health status. For true patriots.
+  components:
+  - type: Sprite
+    sprite: Clothing/Eyes/Hud/medpatch.rsi
+  - type: Clothing
+    sprite: Clothing/Eyes/Hud/medpatch.rsi
+  - type: ShowHealthBars
+    damageContainers:
+    - Biological
+  - type: ShowHealthIcons
+    damageContainers:
+    - Biological
+  - type: Tag
+    tags:
+    - HudMedical
+
+- type: entity
+  parent: ClothingEyesBase
+  id: ClothingEyesEyepatchHudSecurity
+  name: security hud eyepatch
+  description: A heads-up display that scans the humanoids in view and provides accurate data about their ID status and security records. For true patriots.
+  components:
+  - type: Sprite
+    sprite: Clothing/Eyes/Hud/secpatch.rsi
+  - type: Clothing
+    sprite: Clothing/Eyes/Hud/secpatch.rsi
+  - type: ShowSecurityIcons
+  - type: Tag
+    tags:
+    - HudSecurity
+
+- type: entity
+  parent: ClothingEyesBase
+  id: ClothingEyesEyepatchHudBeer
+  name: beer eyepatch
+  description: A pair of sunHud outfitted with apparatus to scan reagents, as well as providing an innate understanding of liquid viscosity while in motion. For true patriots.
+  components:
+  - type: Sprite
+    sprite: Clothing/Eyes/Hud/beerpatch.rsi
+  - type: Clothing
+    sprite: Clothing/Eyes/Hud/beerpatch.rsi
+  - type: ShowThirstIcons
+  - type: StealTarget
+    stealGroup: ClothingEyesHudBeer
```

Comment:
I think you can just position the parents differently so it doesn't look so bulky.

For example, replace the parent of ClothingEyesEyepatchHudMedical with ClothingEyesHudMedical. by doing the same with the other eye patches, you'll reduce the amount of text considerably

Reply:
not wrong